### PR TITLE
Completed prep-javascript-loops exercise.

### PR DESCRIPTION
**Pull request by MehdiHouasse**
Screenshot 2023-06-24 at 4 57 39 PM

Screenshot 2023-06-24 at 4 58 03 PM

### CODE REVIEW

File: prep-javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+// EXERCISE 1
+function whileLoop1() {
+  const Array = [];
```
Comment (BrettAlbright) [inline prep-javascript-loops/main.js]: you should not use capital letters for variables unless they are a specific type of function.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
make sure to change all of the `Array` variables to `array` . Capital letters are not typically used for the first letter in variable names

File: prep-javascript-loops/main.js (hunk 1 above)
Comment (MehdiHouasse) [reply prep-javascript-loops/main.js]: I have fixed the issue, Thank you

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**